app/routers/rddl_network.py

```diff
@@ -0,0 +1,115 @@
+        accountID, sequence, status = getAccountInfo(config.planetmint_api, keys.planetmint_address)
+        print(accountID)
+        print(sequence)
+        if status != "":
+            return {"status": "error", "error": status, "message": status}
+        else:
+            return {"status": "success", "accountinfo": {"accountid": str(accountID), "sequence": str(sequence)}}
+    except Exception as e:
+        return {"status": "error", "error": str(e), "message": str(e)}
+
+
+@router.get("/machine")
+async def getMachineAttestation():
+    try:
+        keys = trust_wallet_instance.get_planetmint_keys()
+        machine_data, status = getMachineInfo(config.planetmint_api, keys.planetmint_address)
+        if status != "":
+            return {"status": "error", "error": status, "message": status}
+        else:
+            return {"status": "success", "machine": machine_data}
+    except Exception as e:
+        return {"status": "error", "error": str(e), "message": str(e)}
+
+
+@router.get("/attestmachine")
+async def getAttestMachine():
+    try:
+
+        keys = trust_wallet_instance.get_planetmint_keys()
+        accountID, sequence, status = getAccountInfo(config.planetmint_api, keys.planetmint_address)
+        additionalCID = ""
+        machineIDSig = computeMachineIDSignature(config.machine_id)
+        print(machineIDSig)
+        gps_data = '{"Latitude":"-48.876667","Longitude":"-123.393333"}'
+        deviceDefinition = '{"Manufacturer": "RDDL","Serial":"AdnT2uyt"}'
+
+        machine_attestation_tx = attestMachine(
+            keys.planetmint_address,
+            "EnergyAgent0",
+            keys.extended_planetmint_pubkey,
+            keys.extended_liquid_pubkey,
+            gps_data,
+            deviceDefinition,
+            config.machine_id,
+            machineIDSig,
+            additionalCID,
+            config.chain_id,
+            accountID,
+            sequence,
+        )
+        response = broadcastTX(machine_attestation_tx)
+
+        if response.status_code != 200:
+            return {"status": "error", "error": response.reason, "message": response.text}
+        else:
+            return {"status": "success", "message": response.text}
+    except Exception as e:
+        return {"status": "error", "error": str(e), "message": str(e)}
+
+
+@router.get("/notarize")
+async def notarize():
+    try:
+
+        keys = trust_wallet_instance.get_planetmint_keys()
+        accountID, sequence, status = getAccountInfo(config.planetmint_api, keys.planetmint_address)
+        notarize_tx = notarizeAsset("cidstr", config.chain_id, accountID, sequence)
+        response = broadcastTX(notarize_tx)
+
+        if response.status_code != 200:
+            return {"status": "error", "error": response.reason, "message": response.text}
+        else:
+            return {"status": "success", "message": response.text}
+    except Exception as e:
+        return {"status": "error", "error": str(e), "message": str(e)}
```

Comment:
this endpoint is not needed because the notarization happens in the data agent class or is it needed for the initial machine attestation